src/logging.cc

```diff
@@ -897,7 +899,9 @@ bool LogFileObject::CreateLogfile(const string& time_pid_string) {
   string string_filename = base_filename_+filename_extension_+
                            time_pid_string;
   const char* filename = string_filename.c_str();
-  int fd = open(filename, O_WRONLY | O_CREAT | O_EXCL, 0664);
+  int fd = open(filename,
+                O_WRONLY | O_CREAT | (!FLAGS_usedatefilenames) ? O_EXCL : 0,
```

Comment:
I'd make the parentheses cover the ternary expression (no need for them around the condition part of it, but I'd like to avoid any confusion on whether `O_WRONLY | O_CREAT |` is part of the ternary condition, or outside of it (I can't remember with certainty priority of operations between ternary and bitwise-or, which I'd take to mean that making it explicit is probably a good idea, even if it's not wrong!).